nuxeo-core/nuxeo-core-convert/src/main/java/org/nuxeo/ecm/core/convert/service/ConversionServiceImpl.java

```diff
@@ -400,10 +414,12 @@ public BlobHolder convertToMimeType(String destinationMimeType, BlobHolder blobH
         String srcMt = blobHolder.getBlob().getMimeType();
```

Comment:
While you're editing this file, is it possible to rename `srcMt` to `srcMimeType` to improve readability ?